boards/ikea-tradfri/include/periph_conf.h

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (C) 2017 Bas Stottelaar <basstottelaar@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_ikea-tradfri
+ * @{
+ *
+ * @file
+ * @brief       Configuration of CPU peripherals for the IKEA TRÅDFRI modules
+ *
+ * @author      Bas Stottelaar <basstottelaar@gmail.com>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "cpu.h"
+
+#include "periph_cpu.h"
+
+#include "em_cmu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Internal macro to calculate *_NUMOF based on config.
+ */
+#define PERIPH_NUMOF(config)    (sizeof(config) / sizeof(config[0]))
+
+/**
+ * @name    Clock configuration
+ * @{
+ */
+#ifndef CLOCK_HF
+#define CLOCK_HF            cmuSelect_HFRCO
+#endif
+#ifndef CLOCK_CORE_DIV
+#define CLOCK_CORE_DIV      cmuClkDiv_1
+#endif
+#ifndef CLOCK_LFA
+#define CLOCK_LFA           cmuSelect_LFRCO
+#endif
+#ifndef CLOCK_LFB
+#define CLOCK_LFB           cmuSelect_LFRCO
+#endif
+#ifndef CLOCK_LFE
+#define CLOCK_LFE           cmuSelect_LFRCO
+#endif
+/** @} */
+
+/**
+ * @brief   RTC configuration
+ */
+#define RTC_NUMOF           (1U)
+
+/**
+ * @name    RTT configuration
+ * @{
+ */
+#define RTT_NUMOF           (1U)
+
+#define RTT_MAX_VALUE       (0xFFFFFFFF)
+#define RTT_FREQUENCY       (1U)
+/** @} */
+
+/**
+ * @name    SPI configuration
+ * @{
+ */
+static const spi_dev_t spi_config[] = {
+    {
+        .dev = USART1,
+        .mosi_pin = GPIO_PIN(PD, 15),
+        .miso_pin = GPIO_PIN(PD, 14),
+        .clk_pin = GPIO_PIN(PD, 13),
```

Comment:
Is there anything connected to these pins (or rather: are they accessible)? If not, we might just want to skip the SPI configuration all together for this board?!

Reply:
There is an additional SPI flash that we might want to access.

Reply:
Alright, so never mind.

Reply:
I'm not 100% sure, but based on reading the JEDEC ID and Product Identification, it seems that a 2Mbit flash by Integrated Silicon Solution Inc. (part number [IS25LQ020B](http://www.issi.com/WW/pdf/25LQ025B-512B-010B-020B-040B.pdf)) is used.